.github/workflows/build-chatbot.yml

```diff
@@ -0,0 +1,47 @@
+name: updater of source links in online book # Name of the workflow
+
+on:
+  push:
+    branches:
+    - main
```

Comment:
Remove main. We don't need to track the unpublished content.

Reply:
Done